webapp/channels/src/components/user_settings/display/user_settings_theme/user_settings_theme.tsx

```diff
@@ -147,18 +146,6 @@ export default class ThemeSetting extends React.PureComponent<Props, State> {
         this.props.setRequireConfirm?.(false);
     };
 
-    handleImportModal = (): void => {
-        this.props.actions.openModal({
-            modalId: ModalIdentifiers.IMPORT_THEME_MODAL,
-            dialogType: ImportThemeModal,
-            dialogProps: {
-                callback: this.updateTheme,
-            },
-        });
-
-        this.props.setEnforceFocus?.(false);
```

Comment:
can you cross check if this prop is used anywhere else in the user_settings_theme component?

Reply:
We do not required it and so I have removed it.